resources/data/dictionary/inputs.csv

```diff
@@ -149,6 +149,8 @@ build_existing_model.water_heater_in_unit,:ref:`water_heater_in_unit`
 build_existing_model.water_heater_location,:ref:`water_heater_location`
 build_existing_model.window_areas,:ref:`window_areas`
 build_existing_model.windows,:ref:`windows`
+build_existing_model.custom_state,:ref:`custom_state`
+build_existing_model.secondary_hvac_heating_type,:ref:`custom_state`

```

Comment:
Wrong ref here. Should also keep this list alphabetized.

Reply:
Done. [919451a](https://github.com/NREL/resstock/pull/1214/commits/919451aa863674522514a541c0b3172d78b6dd65)